src/detectors/RPOTS/RomanPotsReconstruction_factory.h

```diff
@@ -54,17 +55,18 @@ namespace eicrecon {
 	std::string m_readout;
 	std::string m_layerField;
 	std::string m_sectorField;
+	std::string m_geoSvcName;
```

Comment:
It is not critical or a blocker of this PR, but could I kindly suggest that we have coding conventions

https://github.com/eic/EICrecon/blob/main/CONTRIBUTING.md

Reply:
I will need to review this more-carefully to catch everything which doesn't currently conform, will certainly do this for the next PR.